spec/classes/init_spec.rb

```diff
@@ -52,6 +52,26 @@
           it { is_expected.to compile.with_all_deps }
           it { is_expected.to create_iptables_rule('prevent_ipv6_localhost_spoofing').with_apply_to('ipv6') }
         end
+
+        context 'iptables_ports_hash in hiera' do
```

Comment:
Nitpick: Contexts should be labeled so when strung together with their parent `describe` or `context`, and their children expectations, they form a proper sentence.  For example,

```
describe 'foo' do
  context 'when doing a thing' do
    it { is_expected.to return('bar')
  end
end
```

This strings together as `foo when doing a thing is expected to return bar`.